esphome/components/mcp4728/__init__.py

```diff
@@ -0,0 +1,27 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import i2c
+from esphome.const import CONF_ID
+
+DEPENDENCIES = ["i2c"]
+MULTI_CONF = True
+AUTO_LOAD = ["output"]
+
```

Comment:
Please also add yourself as a CODEOWNER and run `script/build_codeowners.py`
```suggestion
CODEOWNERS = ["@brotherdust"]

```